velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -293,11 +294,19 @@ class DecodedVectorTest : public testing::Test, public VectorTestBase {
       SelectivityVector selection(dictionarySize);
       DecodedVector decoded(*dictionaryVector, selection, false);
       check(decoded);
+      auto wrappedVector =
+          decoded.wrap(base, *dictionaryVector, selection.end());
+      // Ensure wrapped vector's values are the same as the original vector's
```

Comment:
This comment is redundant. It just repeats the next line of code. Consider removing.